### PR TITLE
Correct-raw-svg-generation

### PR DESCRIPTION
**Pull request by ggoneiESS**
Separates svg and HTML files from each other so svg can be used directly in applications that can only display images (Grafana)

Replaces functionality removed in 01a2b95acc55c9db1d9aabbb94fa32cd31ace7df that breaks CODA Grafana dashboard

### CODE REVIEW

**Comment by ggoneiESS** [comment]
@tbukovics I've implemented what we discussed, with your better file handling, my 'better' replacements in-place, and a static index.html so all the changes are done only to the svg so it is easily imported to any and all pages/objects/etc.

One nice change is to update the svg not via img, embed, or object, but instead using AJAX to refresh the content.  It means zero blinking since we never actually reload the main page.

**Comment by ggoneiESS** [comment]
and I did it in the 30 minute timer ;)

File: dashboard/generate.py [hunk 1]
```diff
@@ -354,18 +346,27 @@ def run(self):
...
-
+    # edit refresh into index rather than svg, in case of svg issues
+    # file already in place from start-up script
+    with open(f"{args.out}/index.html", "r") as f:
+        newlines = []
+        for line in f.readlines():
+            newlines.append(line.replace("###refresh###", f"{args.refresh * 1000}"))
+    with open(f"{args.out}/index.html", "w") as f:
+        for line in newlines:
+            f.write(line)
```
Comment (tbukovics) [inline dashboard/generate.py]: The webserver is running when this script starts or restarting, so to be nice, maybe you should buffer the lines in one variable and than use the sync_write_fs function I created.

**Review by tbukovics — COMMENTED** [review]
I have one comment to consider

**Review by tbukovics — APPROVED** [review]
I like this, I will shortly inform Morten about the change and then merge.

**Pull request merged**